scripts/common.py

```diff
@@ -59,18 +59,19 @@ def execute(cmd, abort = True):
     status = p.returncode
     if debug:
         message = 'Execution of "{0}" returned with exit code {1}\n'.format(cmd, status)
-        message += '    stdout: "{0}"\n'.format(stdout.decode('ascii').rstrip('\n'))
-        message += '    stderr: "{0}"'.format(stderr.decode('ascii').rstrip('\n'))
+        message += '    stdout: "{0}"\n'.format(stdout.decode('ascii', 'ignore').rstrip('\n'))
+        message += '    stderr: "{0}"'.format(stderr.decode('ascii', 'ignore').rstrip('\n'))
```

Comment:
Normally, I would use argument names for optional arguments:
```
stdout.decode(encoding='ascii', errors='ignore')
```

Reply:
I added the argument names as requested, thanks for the suggestion.